### DISCUSSION TITLE
Propuesta de marco de herramientas para el desarrollo de materiales didácticos

### DISCUSSION THREAD

**Discussion by DavidLMS** [IES-Rafael-Alberti/GT-DVIA-21-22#3] [General]
En este hilo de discusión puedes dar tu opinión sobre la [propuesta de marco de herramientas para el desarrollo de materiales didácticos](https://www.canva.com/design/DAE2n7dnHVc/P6ldmKX0ON6cGlT-CZr8ig/view?utm_content=DAE2n7dnHVc&utm_campaign=designshare&utm_medium=link&utm_source=publishsharelink).

**Comment by csangar034** [comment]
Veo interesantes conocer la herramienta TiddlyWiki y h5p

**Reply by DavidLMS** [reply]
Podemos plantear hacer una formación en ambas herramientas, que además pueden integrarse entre sí.
